### ISSUE TITLE
InnerBlocks Layout Documentation is out of date

### ISSUE THREAD

**Issue by belcherj** [WordPress/gutenberg#7868]
**Describe the bug**
https://github.com/WordPress/gutenberg/tree/master/editor/components/inner-blocks#layouts is out of date given the new nesting in the Columns block

```
<!-- wp:columns -->
<div class="wp-block-columns">
	<!-- wp:paragraph {"layout":"column-1"} -->
	<p>First Paragraph</p>
	<!-- /wp:paragraph -->

	<!-- wp:paragraph {"layout":"column-1"} -->
	<p>Second Paragraph</p>
	<!-- /wp:paragraph -->

	<!-- wp:paragraph {"layout":"column-2"} -->
	<p>Third Paragraph</p>
	<!-- /wp:paragraph -->
</div>
<!-- /wp:columns -->
```

The columns should be nested.  I think this section should be reworked given the changes. Some of what is being described no longer makes sense.

**Comment by chrisvanpatten** [comment]
This will be fixed with #7841.

**Comment by designsimply** [comment]
Closing in favor of #7841. Thank you for posting this!
